src/renderer/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+export interface INotificationProps{
+    isVisible: boolean;
+    level: "info" | "error" | string;
+    horizontalPosition?: "left" | "right" | "center-horizontal" | string;
+    verticalPosition?: "top" | "bottom" | "center-vertical" | string;
+    children?: any;
+    title?: string;
+    onClose: () => void;
+}
+
+export const Notification: React.FunctionComponent<React.PropsWithChildren<INotificationProps>> = (
+    props: INotificationProps) => {
+        
+    return(
+        <div className={`notification-container
+        ${props.level} 
+        ${props.isVisible ? "visible":"none"}
+        ${props.horizontalPosition}
+        ${props.verticalPosition}`}>
```

Comment:
This actually generates new lines and makes html wierd. It would be better to create method that will append css classes and return string